src/ui/components/ConfirmButton/index.js

```diff
@@ -90,23 +91,28 @@ export class ConfirmButtonBase extends React.Component<InternalProps> {
     if (showConfirmation) {
       return (
         <div className={classNames}>
-          <span className="ConfirmButton-message">{message}</span>
-
+          {message && <span className="ConfirmButton-message">{message}</span>}
           <div className="ConfirmButton-buttons">
-            <Button
-              buttonType={confirmButtonType}
-              className="ConfirmButton-confirm-button"
-              onClick={this.onConfirm}
-            >
-              {confirmButtonText || i18n.gettext('Confirm')}
-            </Button>
-            <Button
-              buttonType={cancelButtonType}
-              className="ConfirmButton-cancel-button"
-              onClick={this.toggleConfirmation}
-            >
-              {cancelButtonText || i18n.gettext('Cancel')}
-            </Button>
+            <div className="ConfirmButton-confirm-button-container">
```

Comment:
I don't see this class used anywhere. Do you still need the div?

Reply:
You're right, I think I used them at one point, but they're no longer needed.